cmd/registry/patch/patch_project_test.go

```diff
@@ -174,7 +174,6 @@ func TestProjectExport(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Setup: failed to create client: %+v", err)
 			}
-			defer adminClient.Close()
```

Comment:
I noticed we were getting uncaught errors in test teardowns caused by this `Close()` being called before the `Cleanup()` method, so I moved the `Close()` call inside `Cleanup()`.